.github/workflows/test-module.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test
+on: 
+  workflow_dispatch:
+    inputs:
+      kind-version:
+        required: true
+        default: 'kind19'
+      fybrik-version:
+        required: true
+        default: '0.6.0'
+      module-version:
+        required: true
+        default: '0.6.0'
+      module-type:
+        required: true
+        default: 'stream'
+
+env:
+  GO_VERSION: 1.16
```

Comment:
@Mohammad-nassar10 Please rebase. go version should be now 1.17 (as in build.yaml workflow).